lingui.config.js

```diff
@@ -0,0 +1,10 @@
+/** @type {import('@lingui/conf').LinguiConfig} */
+module.exports = {
+	locales: ["en", "fr"],
+	sourceLocale: "en",
```

Comment:
This is the language that the messages are originally in. This language also will not require any translation work unless you want something changed since it pulls the message straight out into locales/en/messages.po.